crates/jstz_cli/src/deploy.rs

```diff
@@ -41,6 +44,9 @@ pub async fn exec(
 
     let code = read_file_or_input_or_piped(code)?
         .ok_or(user_error!("No function code supplied. Please provide a filename or pipe the file contents into stdin."))?;
+    if code.len() > MAX_CODE_LENGTH {
```

Comment:
`code.len()` doesn't necessarily give the number of bytes, but the number of utf-8 codepoints

Reply:
That's worth fixing! Good spot.